.env.example

```
@@ -82,3 +82,31 @@ MINECRAFT_SERVER_HOST=play.lighthousemc.net
 MINECRAFT_SERVER_NAME="Lighthouse MC"
 MINECRAFT_SERVER_PORT_BEDROCK=19132
 MINECRAFT_SERVER_PORT_JAVA=25565
+
+# Discord Integration
+DISCORD_CLIENT_ID=
+DISCORD_CLIENT_SECRET=
+DISCORD_BOT_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_BOT_TOKEN key should go before the DISCORD_CLIENT_ID key

(UnorderedKey)